### DISCUSSION TITLE
Bazel Run - Fatal error: 'zlib.h' file not found

### DISCUSSION THREAD

**Discussion by steniooliveira** [carbon-language/carbon-lang#1722] [General]
Hi! I'm new here.

Trying to execute command bellow in a distro UBUNTU 20.04. 

Command line:
$ bazel run //explorer -- ./explorer/testdata/print/format_only.carbon

And the return is this:
external/llvm-project/llvm/lib/Support/Compression.cpp:21:10: fatal error: 'zlib.h' file not found
#include <zlib.h>
         ^~~~~~~~
1 error generated.
Target //explorer:explorer failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 22.568s, Critical Path: 19.77s
INFO: 9 processes: 5 internal, 4 linux-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
 
 Any help for this case?

Thx!

**Comment by yknishidate** [comment]
In my case, I installed zlib and the error was resolved. 
```bash
sudo apt install zlib1g-dev
```
See this PR #1558.

**Reply by steniooliveira** [reply]
Hi @yknishidate!
Thx for help. I will try and report soon.

**Comment by steniooliveira** [comment]
![image](https://user-images.githubusercontent.com/20630534/181086996-e5895f9f-50d0-4919-8d11-8df3f8b5683d.png)
Done!

Thx @yknishidate 😃
